### DISCUSSION TITLE
Windows Server 2022 Standard  RDP Black Screen When Connecting

### DISCUSSION THREAD

**Discussion by tamigi** [kimmknight/remoteapptool#123] [Q&A]
windows 10 22h2 client connect to windows server 2022 standard
![11](https://github.com/user-attachments/assets/38f6ed71-1c5a-41cf-aa95-25ea988c3297)

**Comment by kimmknight** [comment]
This appears to be an issue with either application or with RemoteApp, not RemoteApp Tool itself. I will move this to Discussions.

**Comment by tamigi** [comment]
Hello .
I don't have any problem when I connect remote desktop to Windows server 2022 standard, and when I create a remote app with the program, it happens.
